### DISCUSSION TITLE
AdGuardHome.yaml is missing

### DISCUSSION THREAD

**Discussion by Ihtyandr** [AdguardTeam/AdGuardHome#6844] [Q&A]
Hi,
I've been trying to make it work on 2 VMs Ubuntu 22.04. On both, yaml file is just not generated. I am using CLI only, so I am stuck on the first step where I have to put my public IP. 
Is it possible to overcome this issue? 
I tried locating it, grep .yaml - nothing works. It just isn't there

**Comment by ghost** [comment]
If you used the automated installer, you'll find the .yaml generated here by default: `ubuntu@ubuntu-None:~$ sudo ls /opt/AdGuardHome`

**Reply by Ihtyandr** [reply]
That's the thing.. it's not there.

`ubuntu@adguardhome:/opt/AdGuardHome$ sudo ls /opt/AdGuardHome`
`AdGuardHome  AdGuardHome.sig  CHANGELOG.md  LICENSE.txt  README.md  data`

**Reply by ghost** [reply]
I'm not sure then, I ran the automated installer on a fresh Ubuntu vm and it produced the correct result.

Does it happen on other systems or just that one?

**Reply by Ihtyandr** [reply]
I tried on 22.04 Ubuntu and it was the same result for both

**Reply by ghost** [reply]
Have you completed the setup process in the AGH web interface?

Installing via cli is only half of the setup process.

You then have to login to the webui which by default will be 127.0.0.1:3000, or [::1]:3000.

After this step, the yaml will be created.

**Comment by jdoe37** [comment]
You need to "Initial & Setup" first, set http/dns port, username and password...

Only CLI? I am not sure whether there is CLI initial/setup steps.

After SETUP, you will get the yaml file.

**Reply by jdoe37** [reply]
Of course, you also can copy a valid yaml file from other server, restart the service.

**Reply by Ihtyandr** [reply]
Thanks! Do you have one that could be shared here?

**Reply by jdoe37** [reply]
There are some personal data saved in this yaml file(username & PASSWORD). Public sharing is not a good idea. Please use Web UI to finish setup process, get your only yaml and save it for future using.
